content/en/quality_gates/setup/_index.md

```diff
@@ -0,0 +1,187 @@
+3. Select the rule type. You can choose between `Static Analysis` and `Test`.
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline (if the rule is blocking). You can select one of the existing rule conditions for the rule type you have selected.
+
+The following example shows how to create a static analysis rule that fails when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+
+   {{< img src="ci/qg_rule_condition_sa_errors_security_2.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+
+5. Select whether the rule should block the pipeline when it fails.
+   Non-blocking rules are helpful when you roll out a new rule and want to verify its behavior before making it blocking.
+6. Select the time window over which the query is evaluated.
+7. Specify a rule name that describes the rule that you are creating.
+8. Click **Save Rule**.
+
+
+## Invoking quality gates
+
+Quality Gates, requires [`datadog-ci`][7] version `2.19.0` or later.
+
+Invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the rules that have [rule scopes](#rule-scope) and [custom scopes](#custom-scope) that match the current pipeline context (repository, branch, or custom scope passed in the command).
+2. Evaluates all the matching rules.
+3. Fails if one or more blocking rules fail, thereby blocking the pipeline.
+
+<div class="alert alert-danger">For the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
+</div>
+
+| Environment Variables | Description |
+|---|---|
+| `DD_API_KEY` | Point to your [Datadog API key][5]. |
+| `DD_APP_KEY` | Point to your [Datadog application key][6]. |
+| `DD_BETA_COMMANDS_ENABLED` | Set to `true`. |
+| `DD_SITE` | (Optional) Point to a specific [Datadog site][12] (default value is `datadoghq.com`). |
+| LOGGING_MAXDAYS | Number of days to keep file logs on the system before deleting them. Can be any number when running an unattended installation. |
+| WORKERCONFIG_FILEPATH | This should be changed to the path to your Synthetics Private Location Worker JSON configuration file. Wrap this path in quotes if your path contains spaces. |
+
+<pre>
+<code>
+DD_BETA_COMMANDS_ENABLED=true DD_SITE={{< region-param key="dd_site" >}} DD_API_KEY=&lt;API_KEY&gt; DD_APP_KEY=&lt;APP_KEY&gt; datadog-ci gate evaluate
+</code>
+</pre>
```

Comment:
Is there a reason we're using html instead of backticks or codeblock shortcode here? It's not getting the (useful) copy button as a result

Reply:
ah, yes I think that's worth updating - having the copy button would be nice